doc/syntax/operators.rdoc

```diff
@@ -0,0 +1,22 @@
+= Operators
+
+In Ruby, operators such as +++, are defined as methods on the class.
+Literals[link:/literals.rdoc] define their methods within the lower level, C language.
+{String Class}[rdoc-ref:string.c], for example.
+
+Non-literal objects can define or overload their own implementation for a given operator.
```

Comment:
This sentence feels ambiguous.
What do you mean by "non-literal objects" here?
Non-builtin class as the following?
Or, regarding that singleton methods cannot be defined on literals?

Reply:
I was thinking of Ruby objects. You're right, `Non-literal` is very ambiguous. pushed up a fix.